.pr-rubocop.yml

```diff
@@ -0,0 +1,60 @@
+Documentation:
```

Comment:
There's already a `.rubocop.yml` file here https://github.com/moneyadviceservice/frontend/blob/master/.rubocop.yml. 

Do we possibly need to merge these two Rubocop templates so that we have a single source of truth? Or is the special reason for having `.pr-rubocop.yml` alongside `.rubocop.yml`?

Reply:
Apologies - this was accidentally committed, we've been pointed towards using the `rubocop.yml` [here](https://maswiki.valiantyscloud.net/display/NewDev/PR+Approval+Process) run our PRs against.

Reply:
fixed now 👍  @Tr4pSt3R